src/ScottPlot5/ScottPlot/Control/Backend.cs

```diff
@@ -114,6 +114,10 @@ public void TriggerMouseMove(Pixel position)
                     {
                         TriggerMouseDrag(lastMouseDown, position, button);
                     }
+                    else if (Plot.ZoomRectangle.IsVisible)
+                    {
+                        Plot.MouseZoomRectangleClear(applyZoom: false);
```

Comment:
I think this belongs more in `DefaultEventHandlers.OnMouseDown`, I'm trying to keep the logic in this class strictly to delegating to updating state and routing events wherever they ought to go. The actual handling of the events belongs in the event handlers.

Reply:
The reason I did this is because there was a bug where when you re-entered the dead zone the red box would still be drawn. I agree there is probably a better solution than this though.

I'll merge this PR but open a new issue to track this.